pkg/rpc/server/server.go

```diff
@@ -726,7 +726,7 @@ contract_loop:
 		if !ok {
 			cfg := s.chain.GetConfig()
 			if !cfg.P2PStateExchangeExtensions && cfg.RemoveUntraceableBlocks {
-				return nil, response.NewInternalServerError(fmt.Sprintf("failed to get LastUpdatedBlock for balance of %s token", cs.Hash.StringLE()), nil)
+				return nil, response.NewRPCError(fmt.Sprintf("failed to get LastUpdatedBlock for balance of %s token", cs.Hash.StringLE()), "", nil)
```

Comment:
Still an internal error? It can't happen if the node is OK at processing notifications.

Reply:
Yes, fixed.